src/main/java/io/committed/speedy/format/StagedMojo.java

```diff
@@ -12,26 +23,15 @@
 import org.eclipse.jgit.treewalk.filter.PathFilterGroup;
 import org.eclipse.jgit.treewalk.filter.TreeFilter;
 
-import java.io.File;
-import java.io.IOException;
-import java.nio.file.Path;
-import java.util.List;
-import java.util.Set;
-import java.util.stream.Collectors;
-
-import static java.lang.String.format;
-import static java.util.Arrays.asList;
-import static java.util.stream.Collectors.toList;
-
 @Mojo(name = "staged", threadSafe = true)
 public class StagedMojo extends SpotlessApplyMojo {
 
   private static final List<ChangeType> CHANGE_TYPES =
       asList(ChangeType.ADD, ChangeType.COPY, ChangeType.MODIFY, ChangeType.RENAME);
 
   @Override
-  protected void process(List<File> files, Formatter formatter) throws MojoExecutionException {
-    if (files.isEmpty()) {
+  protected void process(Iterable<File> files, Formatter formatter) throws MojoExecutionException {
+    if (files.iterator().hasNext()) {
```

Comment:
We want to return when there are no files to process

```suggestion
    if (!files.iterator().hasNext()) {
```